go/cmd/vtgate/cli/plugin_auth_clientcert.go

```diff
@@ -19,10 +19,19 @@ package cli
 // This plugin imports clientcert to register the client certificate implementation of AuthServer.
 
 import (
+	"github.com/spf13/pflag"
+
 	"vitess.io/vitess/go/mysql"
+	"vitess.io/vitess/go/vt/servenv"
 	"vitess.io/vitess/go/vt/vtgate"
 )
 
+var clientcertAuthMethod string
+
 func init() {
-	vtgate.RegisterPluginInitializer(func() { mysql.InitAuthServerClientCert() })
+	servenv.OnParseFor("vtgate", func(fs *pflag.FlagSet) {
+		fs.StringVar(&clientcertAuthMethod, "mysql_clientcert_auth_method", string(mysql.MysqlClearPassword), "client-side authentication method to use. Supported values: mysql_clear_password, dialog.")
+	})
```

Comment:
This one and others still have the same flags, but now moved into `vtgate` which I think is a better place anyway to have them set up.